Yoast/wordpress-seo
# URL watcher notification changes

## Summary

This pull includes the changes in #10077, but makes some changes:

* Removes the watcher for slug changes as core actually handles that nicely
* Adds branding to the other notifications and improves the messaging
* Makes the notifications dismissible
 
This PR can be summarized in the following changelog entry:

* none needed.

## Relevant technical choices:

* Added an option to the notification class to add yoast branding.

## Test instructions

This PR can be tested by following these steps:

* Delete or trash a post, see notification.
* Change slug on a post, see no notification, also notices core handles it without pain.

## Quality assurance


Fixes https://github.com/Yoast/wordpress-seo-premium/issues/1798
